packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -882,4 +882,11 @@
       <protected>false</protected>
       <shortDescription>No filters available for this query</shortDescription>
     </labels>
+    <labels>
+      <fullName>quantic_NoFiltersAvailableForThisQueryInTheCurrentTab</fullName>
```

Comment:
Is there a maximum name length? :P

Reply:
```suggestion
      <fullName>quantic_NoFilterForCurrentTab</fullName>
```

Reply:
The sky is the limit x)
joke aside. You absolutely right i'll go with your name 👌